client/package.json

```diff
@@ -23,12 +23,14 @@
     "body-parser": "^1.18.3",
     "cookie-parser": "^1.4.3",
     "express-jwt": "^5.3.1",
+    "file-saver": "^2.0.1",
     "ics": "^2.13.2",
     "js-search": "^1.4.2",
     "local-storage": "^1.4.2",
     "moment": "^2.22.2",
     "moment-range": "^4.0.2",
     "nano": "^8.0.1",
+    "net": "^1.0.2",
```

Comment:
I don't think you actually ended up using this `net` library if not please remove it.